storage/kafka.go

```
@@ -40,5 +68,65 @@ func NewKafkaProducer(topic string, brokers []string) Producer {
+		id, _ := primitive.ObjectIDFromHex(event.Data.OrderId)
+
+		//TODO product event for order confirmed
+		confEvent := types.OrderConfirmedEvent{
+			Id:              event.Data.OrderId,
+			Type:            "OrderConfirmed",
+			Source:          "OrderService",
+			SpecVersion:     "1.0",
+			Data:            types.Order{Id: id, Status: "ORDER_CONFIRMED"},
+			DataContentType: "application/json",
+		}
+		k.producer.ProduceOrderConfirmed("order.confirmed", confEvent)
```

golangci-lint (1.62.2)

Error return value of `k.producer.ProduceOrderConfirmed` is not checked

(errcheck)